launcher/jazzer_main.cpp

```diff
@@ -27,6 +27,12 @@
 #include "absl/strings/str_split.h"
 #include "jvm_tooling.h"
 
+#if defined(_ANDROID)
+#define android true
```

Comment:
Please use uppercase names for macros so that they can't clash with local variables.